server/schedulers/utils.go

```diff
@@ -458,3 +490,165 @@ func toHotPeerStatShow(p *statistics.HotPeerStat, kind rwType) statistics.HotPee
 		LastUpdateTime: p.LastUpdateTime,
 	}
 }
+
+// summaryStoresLoad Load information of all available stores.
+// it will filter the hot peer and calculate the current and future stat(rate,count) for each store
+func summaryStoresLoad(
```

Comment:
How about separate it into two functions so that the logic could be more clear?

Reply:
This function just changed its position.

Reply:
I mean tell the logic of handling TiKV/TiFlash apart?